mu-plugins/blocks/table-of-contents/index.php

```diff
@@ -36,7 +36,9 @@ function init() {
  * @return string Returns the block markup.
  */
 function render( $attributes, $content, $block ) {
-	$items = get_headings( get_the_content() );
+	$post = get_post();
+	$post_content = apply_filters( 'the_content', get_the_content( $post ), $post );
```

Comment:
It looks like `the_content` only takes the `$content` as a param, not post. I don't think this breaks anything, but for consistency [with core](https://developer.wordpress.org/reference/hooks/the_content/), it should probably be 

```suggestion
	$post_content = apply_filters( 'the_content', get_the_content( $post ) );
```